doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/examples/ml-example.md

```diff
@@ -0,0 +1,139 @@
+(vm-ml-example)=
+
+# XGBoost-Ray on VMs
```

Comment:
Note the re-branding from this PR:
https://github.com/ray-project/ray/pull/27687/files
There might be a couple more spots that need updates.